packages/at_secondary_server/lib/src/verb/handler/sync_progressive_verb_handler.dart

```diff
@@ -41,44 +41,56 @@ class SyncProgressiveVerbHandler extends AbstractVerbHandler {
     var atCommitLog = await (AtCommitLogManagerImpl.getInstance()
         .getCommitLog(AtSecondaryServerImpl.getInstance().currentAtSign));
     // Get entries to sync
-    var itr = atCommitLog!.getEntries(
+    var commitEntryIterator = atCommitLog!.getEntries(
         int.parse(verbParams[AT_FROM_COMMIT_SEQUENCE]!) + 1,
         regex: verbParams['regex']);
     // Iterates on all the elements in iterator
     // Loop breaks when the [syncBuffer] reaches the limit.
     // and when syncResponse length equals the [AtSecondaryConfig.syncPageLimit]
-    while (itr.moveNext() &&
+    await populateSyncBuffer(syncBuffer, syncResponse, commitEntryIterator);
+
+    response.data = jsonEncode(syncResponse);
+  }
+
+  @visibleForTesting
+  Future<void> populateSyncBuffer(ByteBuffer syncBuffer,
+      List<dynamic> syncResponse, Iterator<dynamic> commitEntryInterator) async {
```

Comment:
Typo in `commitEntryInterator`

Reply:
Rectified as part of [8fb60c0](https://github.com/atsign-foundation/at_server/pull/1246/commits/8fb60c06b05397811f6d46af460042a0115f4e4f)